include/openssl/rsa.h

```diff
@@ -278,6 +278,9 @@ OPENSSL_EXPORT int RSA_private_decrypt(size_t flen, const uint8_t *from,
                                        uint8_t *to, RSA *rsa, int padding);
 
 
+// Salt length matches digest
+#define RSA_PSS_SALTLEN_DIGEST -1
```

Comment:
NP: can you update any comments that refrence salt_len as -1 to use this instead, I found one on line 313 but there might be more.